config/firebase.php

```diff
@@ -122,6 +129,7 @@
                  */
 
                 'default_bucket' => env('FIREBASE_STORAGE_DEFAULT_BUCKET'),
+
```

Comment:
```suggestion
```

Reply:
It's good to follow the conventions, but I would do it in a separate PR or commit as well. I believe the package currently follows Laravel conventions just by accident at the moment 😅